docs/reference/cat/indices.asciidoc

```diff
@@ -77,6 +79,8 @@ primary shards. Defaults to `false`.
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-s]
 
+include::{docdir}/rest-api/common-parms.asciidoc[tag=time]
+
```

Comment:
I think this one should include bytes too
```
GET _cat/indices?h=ftt,ss&bytes=g&time=ms
```

Reply:
It looks like `bytes` is already documented for this API:
https://github.com/elastic/elasticsearch/blame/be233f3d235b39d9315fc79974e9b69f20b1fe38/docs/reference/cat/indices.asciidoc#L48